test/index.js

```diff
@@ -1345,6 +1357,22 @@ describe('def', function() {
            'Since there is no type of which all the above values are members, the type-variable constraint has been violated.\n');
   });
 
+  it('supports "strict" record types', function() {
+
```

Comment:
For consistency, please remove this empty line and the one before the closing `});`.